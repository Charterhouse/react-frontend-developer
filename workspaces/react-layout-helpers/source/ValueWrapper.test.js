import React from 'react'
import { render } from 'react-testing-library'
import { ValueWrapper } from './'

describe('ValueWrapper', () => {
  it('has proper styling', () => {
    const { container } = render(<ValueWrapper />)
    expect(container).toMatchSnapshot()
  })
})
