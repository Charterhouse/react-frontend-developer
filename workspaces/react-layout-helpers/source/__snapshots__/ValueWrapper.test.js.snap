// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ValueWrapper has proper styling 1`] = `
.emotion-0 {
  border-radius: 15px;
  border: 1px solid #ff80c3;
  padding: 10px;
  margin: 10px;
  min-width: 6rem;
  text-align: center;
}

<div>
  <div
    class="emotion-0 emotion-1"
  />
</div>
`;
